lib/sessionStore.js

```diff
@@ -136,15 +136,17 @@ SessionStore.prototype.destroy = function(sid, callback) {
     try {
         //console.log('SessionStore.destroy ' + sid);
         this.removeSid(sid);
-        callback(null);
+        callback && callback(null);
```

Comment:
so the callback wasn't optional, but now it is, right?

Reply:
Yes - apparently nobody was calling it at all before.